.github/workflows/build.yml

```diff
@@ -64,6 +64,12 @@ jobs:
       - name: Publish Bicep
         run: dotnet publish --configuration ${{ matrix.configuration }} --self-contained true -p:PublishTrimmed=true -p:PublishSingleFile=true -r ${{ matrix.rid }} ./src/Bicep.Cli/Bicep.Cli.csproj
 
+      - name: Run Bicep E2E Tests
```

Comment:
>name [](start = 8, length = 4)

Is it possible to also upload coverage data from this npm run as well?

Reply:
It is possible and easy to do so because we are using jest, but does it make sense to do so? We are testing a binary file and the coverage will be 0 for sure (unless we want to cheat by including the test files themselves :p).

Reply:
We're kind of all over the place in that regard. (c# info includes test code as well). I'm fine skipping that for now.

Reply:
In the future we could split and tag test code coverage separately. It sometimes can be useful to see coverage highlights there, but it's pretty rare.